### PR TITLE
feat: wip enhance apigee edge and opdk reporting

### PR DESCRIPTION
**Pull request by g-devpat**
## Description
What's changed, or what was fixed?

Added capability to report the following Apigee Edge/OPDK resources:
- [ ] Target Servers
- [ ] Keystores
- [ ] API Products
- [ ] Developer Apps
- [ ] Developers
- [ ] Virtual Hosts
- [ ] KVM

## Housekeeping
(please check all that apply [x], do not edit the text)
- [ ] I have run all the tests locally and they all pass.
- [ ] I have followed the relevant style guide for my changes.
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

File: tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+echo "<thead class=\"thead-dark\"><tr>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"id\">Product Name</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"envs\">Environments</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"proxies\">Proxies</th>" >> "$report_html"
+echo "<th data-sortable=\"true\" data-field=\"enabled\">Aproval Type</th>" >> "$report_html"
+echo "</tr></thead>" >> "$report_html"
+
+echo "<tbody class=\"mdc-data-table__content\">" >> "$report_html"
+
+jq -c '.[]' "$export_folder/$organization/config/resources/edge/env/$environment/api-products".json | while read i; do
```
Comment (g-greatdevaks) [inline tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh:24]: Need to validate if the JSON body is having the same structure for OPDK/Edge and X/Hybrid.
Currently, the logic is only being executed for OPDK; not even Edge.

File: tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
+echo "<h3>API Products</h3>" >> "$report_html"
```
Comment (danistrebel) [inline tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh:1]: please include license headers on all new .sh files

Comment (g-greatdevaks) [reply tools/apigee-sackmesser/cmd/report/opdk/apiproducts.sh:17]: Added, Daniel.

**Review by danistrebel — APPROVED** [review]
Great stuff, thanks team! 🎉

**Pull request merged**